SiteConfig: fix rendering issues (#7209)
open-cluster-management/rhacm-docs: mce_acm_integration/siteconfig/siteconfig_scale_out_worker_nodes.adoc

[#scale-out-annotation]
== Adding a worker node

> Add a worker node by updating your `ClusterInstance` CR that was used to provision your cluster.

Reviewer: ```suggestion
Add a worker node by updating your `ClusterInstance` custom resource that is used to provision your cluster.
```

Complete the following steps to add a worker node to the managed cluster:

> . Define a new node object in the existing `ClusterInstance` CR:

Reviewer: ```suggestion
. Define a new node object in the existing `ClusterInstance` custom resource:
```

...
----

> . Apply the changes.

Reviewer: ```suggestion
. Apply the following changes.
```

Author: We actually have something similar in scale in:
`. Apply the changes. See the following options:`

So I added the second sentence instead

.. If you are using GitOps ZTP, push to your Git repository and wait for Argo CD to synchronize the changes.

> . Verify that the `BareMetalHost` resources are added by running the following command on the hub cluster:

Reviewer 2: ```suggestion
. Verify that a new `BareMetalHost` resource is added by running the following command on the hub cluster:
```

+
[source,terminal]
----
oc get bmh -n <clusterinstance_namespace> --watch --kubeconfig <hub_cluster_kubeconfig_filename>
----

+
See the following example output:

+
[source,terminal]
----
NAME                        STATE          CONSUMER   ONLINE   ERROR   AGE
master-node1.example.com    provisioned               true             81m
worker-node2.example.com    provisioning              true             44m
----

> . Verify that the `Agent` resources are added by running the following command on the hub cluster:

Reviewer 2: ```suggestion
. Verify that a new `Agent` resource is added by running the following command on the hub cluster:
```

----
oc get agents -n <clusterinstance_namespace> --watch --kubeconfig <hub_cluster_kubeconfig_filename>
----

+
See the following example output:

+
[source,terminal]
----
NAME                       CLUSTER                   APPROVED    ROLE     STAGE
master-node1.example.com   <managed_cluster_name>    true        master   Done
master-node2.example.com   <managed_cluster_name>    true        master   Done
master-node3.example.com   <managed_cluster_name>    true        master   Done
worker-node1.example.com   <managed_cluster_name>    false       worker 
worker-node2.example.com   <managed_cluster_name>    true        worker   Starting installation
worker-node2.example.com   <managed_cluster_name>    true        worker   Installing
worker-node2.example.com   <managed_cluster_name>    true        worker   Writing image to disk
worker-node2.example.com   <managed_cluster_name>    true        worker   Waiting for control plane
worker-node2.example.com   <managed_cluster_name>    true        worker   Rebooting
worker-node2.example.com   <managed_cluster_name>    true        worker   Joined
worker-node2.example.com   <managed_cluster_name>    true        worker   Done
----

> . Verify that the `Node` resources are added by running the following command on the managed cluster:

Reviewer 2: ```suggestion
. Verify that a new `Node` resource is added by running the following command on the managed cluster:
```

. Verify that the `Node`, `Agent`, and `BareMetalHost` resources are created.

> .. Verify that the `Node` resources are removed by running the following command on the hub cluster:

Reviewer: I recommend that you revert the change here. There should be a new line between the step and plus sign

Author: It doesn't look right and I'm checking the preview in my local editor. The only way to resolve the issue of the codeblocks is by removing the extra space.
![Screenshot from 2024-11-01 17-05-50](https://github.com/user-attachments/assets/2d82e821-4385-4b74-a88f-66c282c20d34)

![Screenshot from 2024-11-01 13-16-05](https://github.com/user-attachments/assets/7d9bb394-02e7-4742-aaed-d01afbefa528)

Author: Anyway, I had to change this section but there are other steps where it's not rendering right:
![Screenshot from 2024-11-01 17-08-31](https://github.com/user-attachments/assets/58aca24a-9de6-4104-8817-5bf4224ba726)

Reviewer: Ok, let's leave it for now and. we can revisit in a refresh
